src/test/java/org/broadinstitute/hellbender/utils/UtilsUnitTest.java

```diff
@@ -1,26 +1,25 @@
 package org.broadinstitute.hellbender.utils;
 
 
+import static java.util.Arrays.asList;
+import static org.testng.Assert.assertEquals;
```

Comment:
No changes required in this file anymore...

Reply:
yes, undone.